hail/python/hail/methods/statgen.py

```diff
@@ -1833,7 +1833,7 @@ def split_multi_hts(ds, keep_star=False, left_aligned=False, vep_root='vep'):
         PID: str,
       }
 
-    For other entry fields, use :class:`.SplitMulti`.
+    For other entry fields, use :func:`.split_multi`.
```

Comment:
this isn't true anymore, I don't think.

Reply:
what isn't true?

Reply:
you mean it should be "annotate_entries"?

Reply:
sorry--I meant that I think that line is from when SplitMulti was a class that could do things to the entries. Probably makes more sense to remove it?

Reply:
yeah